src/components/AirdropWalletsCarouselItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import { CONST, AirdropWalletDetails, AirdropGoal } from 'app/consts';
+
+import { AirdropWalletBalance } from './AirdropWalletBalance';
+import { AirdropWalletFooter } from './AirdropWalletFooter';
+
+const i18n = require('../../loc');
+
+const airdropGoals: AirdropGoal[] = [
+  { threshold: 5, name: i18n.airdrop.walletsCarousel.shrimp },
+  { threshold: 25, name: i18n.airdrop.walletsCarousel.crab },
+  { threshold: 100, name: i18n.airdrop.walletsCarousel.shark },
+  { threshold: 1000, name: i18n.airdrop.walletsCarousel.whale },
+];
+
+const readableOrder = [i18n.order.first, i18n.order.second, i18n.order.third, i18n.order.fourth];
+
+interface Props {
+  walletDetails: AirdropWalletDetails;
+}
+
+export const AirdropWalletsCarouselItem: FC<Props> = ({ walletDetails }) => {
```

Comment:
shouldn't be named AirdropWalletsCarouselItem as it not only used as an item but also in the airdrop wallet details

Reply:
done